techniques/general/control-orientation.html

```diff
@@ -0,0 +1,54 @@
+<!DOCTYPE html>
+<html xmlns="http://www.w3.org/1999/xhtml" lang="en">
+	<head>
+		<title>Using a control to allow access to content in different orientations which is otherwise restricted</title>
+		<link rel="stylesheet" type="text/css" href="https://rawgit.com/w3c/wcag21/master/css/sources.css"/>
+	</head>
+	<body>
+		<h1>Using a control to allow access to content in different orientations which is otherwise restricted</h1>
+		<section id="meta">
+			<h2>Metadata</h2>
+			<p id="id">ID: @@G210</p>
+			<p id="technology">Technology: General</p>
+			<p id="type">Type: Technique</p>
+		</section>
+		<section id="applicability">
+			<p>When the orientation of the page is locked, provide a button to allow a user to change the orientation.</p>
+		</section>
+		<section id="description">
+			<h2>Description</h2>
+			<p>The objective of this technique is to allow users to access content whether their device is oriented in portrait or landscape view. A content provider may expect that most users will view content using a specific orientation or may expect that a user will want to maintain the original view within the device, and as a result the provider prevents the content from rotating in response to device information. By providing a control to allow the user to rotate the content a user who uses their device in a fixed orientation, such as a user who cannot hold the device and has it mounted attached to a wheelchair or bed, will be able to view the content in a comfortable manner.</p>
```

Comment:
Readability suggestion: The objective of this technique is to allow users to access content whether their device is oriented in portrait or landscape view. A content provider may expect that most users will view content using a specific orientation or may expect that a user will want to maintain the original view within the device. As a result the provider then prevents the content from rotating. By providing a control to allow the user to rotate the content, someone who needs to use a particular orientation will be able to view the content in a comfortable manner. For example, a person who cannot hold the device and has it mounted attached to a wheelchair or bed.